class-sc-wp-updates-notifier.php

```diff
@@ -259,6 +266,154 @@ public function plugin_action_links( $links, $file ) {
 		}
 
 
+		/**
+		 * Alter the columns on the plugins page to show enable and disable notifications.
+		 *
+		 * @param string $column_name Name of the column.
+		 * @param string $plugin_file Path to the plugin file relative to the plugins directory.
+		 * @param array  $plugin_data An array of plugin data.
+		 */
+		public function manage_plugins_custom_column( $column_name, $plugin_file, $plugin_data ) {
+			$options = $this->get_set_options( self::OPT_FIELD ); // get settings
+			if ( 1 === $options['notify_plugins'] ) {
+				if ( 'update_notifications' === $column_name ) {
+					if ( is_plugin_active( $plugin_file ) ) {
+						if ( isset( $options['disabled_plugins'][ $plugin_file ] ) ) {
+							echo '<button class="sc_wpun_btn sc_wpun_btn_disable" data-toggle="enable" data-file="' . esc_html( $plugin_file ) . '">' . esc_html( __( 'Notifications Disabled', 'wp-updates-notifier' ) ) . '</button>';
+						} else {
+							echo '<button class="sc_wpun_btn sc_wpun_btn_enable" data-toggle="disable" data-file="' . esc_html( $plugin_file ) . '">' . esc_html( __( 'Notifications Enabled', 'wp-updates-notifier' ) ) . '</button>';
+						}
+					}
+				}
+			}
+		}
+
+
+		/**
+		 * Alter the columns on the plugins page to show enable and disable notifications.
+		 *
+		 * @param array $column_headers An array of column headers.
+		 */
+		public function manage_plugins_columns( $column_headers ) {
+			$options = $this->get_set_options( self::OPT_FIELD ); // get settings
+			if ( 1 === $options['notify_plugins'] ) {
+				$column_headers['update_notifications'] = __( 'Update Notifications', 'wp-updates-notifier' );
+			}
+			return $column_headers;
+		}
+
+
+		/**
+		 * Custom css for the plugins.php page.
+		 */
+		public function custom_admin_css() {
+			$options = $this->get_set_options( self::OPT_FIELD ); // get settings
+			if ( 1 === $options['notify_plugins'] ) {
+				echo '<style type="text/css">
+
+				.column-update_notifications{
+					width: 15%;
+				}
+
+				.sc_wpun_btn:before {
+					font-family: "dashicons";
+					display: inline-block;
+					-webkit-font-smoothing: antialiased;
+					font: normal 20px/1;
+					vertical-align: top;
+					margin-right: 5px;
+					margin-right: 0.5rem;
+				}
+
+				.sc_wpun_btn_enable:before {
+					content: "\f12a";
+					color: green;
+				}
+
+				.sc_wpun_btn_disable:before {
+					content: "\f153";
+					color: red;
+				}
+
+				.sc_wpun_btn_enable:hover:before {
+					content: "\f153";
+					color: red;
+				}
+
+				.sc_wpun_btn_disable:hover:before {
+					content: "\f12a";
+					color: green;
+				}
+
+				</style>';
+			}
+		}
+
+
+		/**
+		 * Custom js for the plugins.php page.
+		 */
+		public function custom_admin_js() {
+			?>
+			<script type="text/javascript" >
```

Comment:
It would be ideal to just include the JS and CSS only on the plugins page versus on all admin pages.  See if you can use the `global $pagenow;` variable to narrow this down.